spec/models/immunity_detail_spec.rb

```diff
@@ -44,4 +44,76 @@
       expect(immunity_detail.evidence_groups.first.tag).to eq("council_tax_document")
     end
   end
+
+  describe "#evidence_gaps?" do
```

Comment:
Might be overkill but would a test for the case where there's no evidence at all be useful? (I think it returns nil then.)